src/main/java/io/confluent/kafkarest/serializers/KafkaJsonSerializer.java

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright 2015 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ **/
+package io.confluent.kafkarest.serializers;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.kafka.common.errors.SerializationException;
+import org.apache.kafka.common.serialization.Serializer;
+
+import java.util.Map;
+
+/**
+ * Serialize JsonNode to UTF-8 JSON.
+ */
+public class KafkaJsonSerializer implements Serializer<Object> {
```

Comment:
Would it make sense to just move the serializer implementations into a package in schema-registry, along with the Avro ones? Obviously they're pretty trivial right now and won't get any real schema registry support unless we add JSON support to the schema registry (in which case we'd need to figure out how to pass along that schema info anyway), but is it better to just house them there and depend on them here?